modules/ROOT/pages/activity-tracking-edit.adoc

```diff
@@ -0,0 +1,58 @@
+= Activity Tracking
+
+B2B transactions often communicate contractual obligations between companies. Due to the sensitivity of this information, it is desirable (and often necessary) to track transmissions and store both transmission content and metadata.
+
+Anypoint Partner Manager enables you to monitor trading partner transmission activity. Activity is the complete set of end-to-end events related to transmission metadata, messages, and errors that are involved in processing a transmission.
+
+Metadata includes things like the date and time the transmission was sent or received, the identity of the partner, and the success or failure of processing.
+
+Common uses for this metadata include:
+
+* Nonrepudiation
+* Auditing
+* Correlation of acknowledgments
+* Error reporting
+
+With activity tracking, you can:
+
+* Search for message metadata (such as partner name, message type, filename, control number, endpoint, or date received).
```

Comment:
Repeats much of what is defined above for metadata, and adds to it; ditto re. consider separation of details into own topic.